src/main.py

```
@@ -0,0 +1,191 @@
+from qgis.PyQt.QtWidgets import (
+    QFileDialog,
+    QMessageBox,
+    QAction,
+    QPushButton,
+    QDialog,
+    QVBoxLayout,
+    QLabel,
+    QCheckBox,
+)
+from qgis.utils import iface
+from .common import *
```

Ruff (0.8.2)

`from .common import *` used; unable to detect undefined names

(F403)